tasks/cron-task

```diff
@@ -1,3 +1,2 @@
-
 # run participate every minute
-*  *  *  *  *   bash ~/app/scripts/validationAndPreparation.sh
\ No newline at end of file
+*  *  *  *  *  bash /app/scripts/validationAndPreparation.sh >> /var/log/cron.log 2>&1
```

Comment:
Given this runs once every minute, what happens if two people submit a response based on the same challenge in a short period of time?

Would it be somehow possible to guarantee that we only take the first response, transform it and then the second response should no longer check out.

Reply:
Good remark: 
Yes, I am currently preparing a pull request so that not two cron jobs are running simultaneously:
https://stackoverflow.com/questions/2366693/run-cron-job-only-if-it-isnt-already-running

If this is guaranteed the second submission will not fit to the updated challenge-file

Reply:
I have implemented a logic such that not two cron jobs can be run simultaneously via the validation_controller.sh